### PR TITLE
MOSIP-32046 : Seed data changes for master

### PR DESCRIPTION
**Pull request by GOKULRAJ136**

### CODE REVIEW

**Review by dhanendra06 — APPROVED** [review]

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql [hunk 1]
```diff
@@ -1 +1,17 @@
 \echo 'Upgrade Queries not required for transition from $CURRENT_VERSION to $UPGRADE_VERSION'
```
Comment (ase-101) [inline db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:1]: We can remove this statement as we have upgrade queries to run.

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql [hunk 2]
```diff
@@ -1 +1,17 @@
...
+
+-- Rollback script for master.app_detail
+ALTER TABLE master.app_detail ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.app_detail DROP CONSTRAINT pk_appdtl_id;
```
Comment (ase-101) [inline db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:3]: We only drop the constraint, we should also add the PK constraint with the lang_code column back @dhanendra06

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql [hunk 3]
```diff
@@ -1 +1,17 @@
...
+
+-- Rollback script for master.biometric_attribute
+ALTER TABLE master.biometric_attribute ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.biometric_attribute DROP CONSTRAINT pk_bmattr_code;
```
Comment (ase-101) [inline db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:7]: We only drop the constraint, we should also add the PK constraint with the lang_code column back @dhanendra06

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql [hunk 4]
```diff
@@ -1 +1,17 @@
...
+
+-- Rollback script for master.module_detail
+ALTER TABLE master.module_detail ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.module_detail DROP CONSTRAINT pk_moddtl_code;
```
Comment (ase-101) [inline db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:11]: We only drop the constraint, we should also add the PK constraint with the lang_code column back @dhanendra06

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql [hunk 5]
```diff
@@ -1 +1,17 @@
...
+
+-- Rollback script for master.template_file_format
+ALTER TABLE master.template_file_format ALTER COLUMN lang_code SET NOT NULL;
+ALTER TABLE master.template_file_format DROP CONSTRAINT pk_tffmt_code;
```
Comment (ase-101) [inline db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:15]: We only drop the constraint, we should also add the PK constraint with the lang_code column back @dhanendra06

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql (hunk 1 above)
Comment (GOKULRAJ136) [reply db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql]: removed as per the comment

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql (hunk 3 above)
Comment (GOKULRAJ136) [reply db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:8]: Added the PK constraint with the lang_code column back

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql (hunk 4 above)
Comment (GOKULRAJ136) [reply db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:13]: Added the PK constraint with the lang_code column back

File: db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql (hunk 5 above)
Comment (GOKULRAJ136) [reply db_upgrade_scripts/mosip_master/sql/1.2.0.1_to_1.2.1.0_rollback.sql:18]: Added the PK constraint with the lang_code column back

**Review by ase-101 — APPROVED** [review]

**Comment by ase-101** [comment]
@dhanendra06 - pls check DB test precheck is failing https://github.com/mosip/admin-services/actions/runs/10364838130/job/28777733119?pr=1048

**Comment by GOKULRAJ136** [comment]
> @dhanendra06 - pls check DB test precheck is failing https://github.com/mosip/admin-services/actions/runs/10364838130/job/28777733119?pr=1048

The issue was due to a foreign key constraint in the template table related to lang_code. It has been resolved and the new changes have been committed.

**Review by ase-101 — APPROVED** [review]

**Pull request merged**